SimCalorimetry/EcalSimAlgos/interface/EcalTDigitizer.h

```diff
@@ -27,9 +27,9 @@ class EcalTDigitizer {
   typedef typename Traits::DigiCollection DigiCollection;
   typedef typename Traits::EcalSamples EcalSamples;
 
-  EcalTDigitizer<Traits>(EcalHitResponse* hitResponse, ElectronicsSim* electronicsSim, bool addNoise);
+  EcalTDigitizer(EcalHitResponse* hitResponse, ElectronicsSim* electronicsSim, bool addNoise);
```

Comment:
@iarspider , template argument for this class is really used by consume class, I would not change this line.

Reply:
@civanch the change is purely syntaxic ([CWG Issue 2337](https://cplusplus.github.io/CWG/issues/2237.html) ) - the `EcalTDigitizer` is still a template, the `Traits` typename is still available to all the functions inside this.

Reply:
OK